MekHQ/data/scenariotemplates/Air Support.xml

```diff
@@ -0,0 +1 @@
UnitCount><forceAlignment>2</forceAlignment><forceMultiplier>1.0</forceMultiplier><forceName>Primary Opfor</forceName><generationMethod>1</generationMethod><generationOrder>5</generationOrder><maxWeightClass>4</maxWeightClass><minWeightClass>1</minWeightClass><retreatThreshold>50</retreatThreshold><startingAltitude>0</startingAltitude><syncDeploymentType>None</syncDeploymentType><useArtillery>false</useArtillery></value></entry><entry><key>Allies</key><value><actualDeploymentZone>-1</actualDeploymentZone><allowAeroBombs>false</allowAeroBombs><allowedUnitType>-2</allowedUnitType><arrivalTurn>0</arrivalTurn><canReinforceLinked>false</canReinforceLinked><contributesToBV>true</contributesToBV><contributesToMapSize>true</contributesToMapSize><contributesToUnitCount>true</contributesToUnitCount><deployOffboard>false</deployOffboard><deploymentZones><deploymentZone>10</deploymentZone></deploymentZones><destinationZone>4</destinationZone><fixedUnitCount>-1</fixedUnitCount><forceAlignment>1</forceAlignment><forceMultiplier>1.0</forceMultiplier><forceName>Allies</forceName><generationMethod>3</generationMethod><generationOrder>4</generationOrder><maxWeightClass>4</maxWeightClass><minWeightClass>1</minWeightClass><retreatThreshold>50</retreatThreshold><startingAltitude>0</startingAltitude><syncDeploymentType>None</syncDeploymentType><useArtillery>false</useArtillery></value></entry></scenarioForces><scenarioObjectives><scenarioObjective><associatedForceNames><associatedForceName>Allies</associatedForceName></associatedForceNames><associatedUnitIDs/><successEffects><successEffect><effectType>ScenarioVictory</effectType><effectScaling>Fixed</effectScaling><howMuch>2</howMuch></successEffect></successEffects><failureEffects><failureEffect><effectType>ScenarioDefeat</effectType><effectScaling>Fixed</effectScaling><howMuch>1</howMuch></failureEffect></failureEffects><additionalDetails/><description>Preserve 50% of the following force(s) and unit(s):</description><destinationEdge>NONE</destinationEdge><objectiveCriterion>Preserve</objectiveCriterion><percentage>50</percentage><timeLimitAtMost>true</timeLimitAtMost><timeLimitType>None</timeLimitType></scenarioObjective><scenarioObjective><associatedForceNames><associatedForceName>Primary Opfor</associatedForceName></associatedForceNames><associatedUnitIDs/><successEffects><successEffect><effectType>ScenarioVictory</effectType><effectScaling>Fixed</effectScaling><howMuch>2</howMuch></successEffect></successEffects><failureEffects><failureEffect><effectType>ScenarioDefeat</effectType><effectScaling>Fixed</effectScaling><howMuch>1</howMuch></failureEffect></failureEffects><additionalDetails/><description>Destroy 50% of the following force(s) and unit(s):</description><destinationEdge>NONE</destinationEdge><objectiveCriterion>ForceWithdraw</objectiveCriterion><percentage>50</percentage><timeLimitAtMost>true</timeLimitAtMost><timeLimitType>None</timeLimitType></scenarioObjective></scenarioObjectives></ScenarioTemplate>
```

Comment:
Not a big deal but if these XML files were formatted to be on multiple lines, diffs going forward will be easier.

Reply:
I recall having some trouble getting JAXB to render linebreaks properly, I'll take another look at it for the next PR.